modules/alma/lib/AlmaClient/AlmaClient.class.php

```diff
@@ -92,6 +92,14 @@ public function request($method, $params = array(), $check_status = TRUE) {
           case 'reservationNotFound':
             throw new AlmaClientReservationNotFound('Reservation not found');
 
+          case 'invalidPatron':
+            if ($method == 'patron/selfReg') {
+              throw new AlmaClientUserAlreadyExistsError();
+            }
+            else {
+              throw new AlmaClientInvalidPatronError('Invailed patron');
```

Comment:
Typo - "invailed".

Also: The string does not seem to provide any additional info. Can it be left out?